File: src/PDFRest.Docs/www/docs/pdfa/payloads/pdfa.js
var formData = new FormData();

formData.append('file', pdfFile);
formData.append('conformanceLevel', 'PDF_A_2B');
formData.append('title', 'My Title');
formData.append('author', 'Author name');
formData.append('creationDate', new Date());
formData.append('customProperties[0].name', 'MyProperty');
formData.append('customProperties[0].value', 'My Property Value');

axios({
  method: 'post',
  url: '/pdfa',
  data: formData,
  config: {
    headers: {
      'Content-Type': 'multipart/form-data'
    }
  }
}).then(function (response) {
  // handle success
}).catch(function (response) {
  // handle error
});

File: src/PDFRest.Docs/www/sections/pdfa/payloads/pdfa.cs
var bytes = File.ReadAllBytes("/path/to/file.pdf");
var pdfFile = new ByteArrayContent(bytes);

pdfFile.Headers.ContentType = new MediaTypeHeaderValue(MediaTypeNames.Application.Pdf);

var formData = new MultipartFormDataContent();
formData.Add(pdfFile, "file", "file");
formData.Add("PDF_A_2B", "conformanceLevel");
formData.Add("My Title", "title");
formData.Add("Author name", "author");
formData.Add(new DateTime().ToString("o"), "creationDate");
formData.Add("MyProperty", "customProperties[0].name");
formData.Add("My Property Value", "customProperties[0].value");

var client = new HttpClient();
var response = await _client.PostAsync("/pdfa", formData);
